applications/istio/bookinfo/README.md

```diff
@@ -0,0 +1,130 @@
+# Istio Bookinfo demo in OAM way
+This application demonstrates how OAM bootstraps [Istio Bookinfo application](https://istio.io/latest/docs/examples/bookinfo/) with workload `Gateway`, `VirtualService`,
+`DestinationRule` deployed by OAM.
+
+## Prerequisites
+- Helm v3
+
+- OAM Kubernetes Runtime
+
+```
+kubectl create namespace crossplane-system
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install addon-oam-kubernetes-local --namespace crossplane-system crossplane-alpha/oam-core-resources
+```
+
+- OAM workloads/traits definitions
+
+```
+$ kubectl apply -f https://raw.githubusercontent.com/crossplane/crossplane/release-0.12/docs/snippets/run/definitions.yaml
+workloaddefinition.core.oam.dev/containerizedworkloads.core.oam.dev created
+traitdefinition.core.oam.dev/manualscalertraits.core.oam.dev created
+workloaddefinition.core.oam.dev/postgresqlinstancerequirements.database.example.org created
+
+```
+
+(TODO)  ** The root cause why this depends on installation of Crossplane has to be investigated.**
```

Comment:
If this issue cannot be solved in recent time, just ask users to install crossplane and avoid the confusion

Reply:
@ryanzhang-oss If we don't install Crossplane (`helm install crossplane --namespace crossplane-system crossplane-alpha/crossplane`), we will hit issue as below when applying  `WorkloadDefinition` and `TraitDefinition` manifests.